lib/kitchen_sink/integer.ex

```diff
@@ -0,0 +1,16 @@
+defmodule KitchenSink.Integer do
+  @moduledoc """
+  this module is for Integer functions
+  """
+
+  @doc """
+  Make sure that we get an integer either from an integer or string
+
+  iex> ensure_integer("1")
+  1
+  iex> ensure_integer(2)
+  2
+  """
+  def ensure_integer(i) when is_integer(i), do: i
```

Comment:
Nice idea behind `LazyInteger`! At a quick glance @olafura, it seems to be Ecto-specific…